### PR TITLE
Added more visibility for movement pruning docs

### PR DESCRIPTION
**Pull request by ljaljushkin**
### Changes

Added movement pruning to the table with compression algorithms.
Also extended the list of supported models.

### Reason for changes

No mentions the algorithm in the main readme.

### Related tickets

90138

### Tests

n/a

### CODE REVIEW

**Review by vuiseng9 — COMMENTED** [review]
lgtm

**Comment by ljaljushkin** [comment]
run tensorflow pre-commit tests

**Review by alexsu52 — APPROVED** [review]
LGTM

File: nncf/experimental/torch/sparsity/movement/MovementSparsity.md [hunk 1]
```diff
@@ -50,14 +50,14 @@ Please refer [optimum-intel](https://github.com/huggingface/optimum-intel/tree/m
 #### Known Limitation
 
 1. Movement sparsification only supports `torch.nn.Linear` layers.
-2. Automatic structured masking feature only supports **_BERT, Wav2Vec2 and Swin_**.
+2. Automatic structured masking feature supports **BERT, DistilBERT, RoBERTa, MobileBERT, Wav2Vec2, Swin, ViT, CLIPVisual** architectures defined by [Hugging Face&#39;s transformers](https://huggingface.co/docs/transformers/index). Other similar architectures may work, but support is not guaranteed.
```
Comment (AlexKoff88) [inline nncf/experimental/torch/sparsity/movement/MovementSparsity.md:53]: A minor comment about `CLIPVisual`. I am not sure what it means. Please rephrase.

**Review by AlexKoff88 — APPROVED** [review]

**Review by AlexKoff88 — APPROVED** [review]

**Review by AlexKoff88 — APPROVED** [review]

**Pull request merged**